### DISCUSSION TITLE
debug.ts

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#2472] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/113572850/221369802-367c9840-23a6-423e-9faa-7adc5baba892.png)
如果一个 id 有许多篇文章，没法一个个加，应该把这两行代码改成什么？
@banned-historical-archives

**Comment by banned-historical-archives** [comment, marked as answer]
for (const i of await book.parser(book.path, book.parser_option)) {
    console.log(get_article_id(i));
  }
